src/flock_server/keybase_notifications.py

```diff
@@ -126,9 +162,22 @@ def add(self, notification, details):
             keybase_notification.save()
 
     def format(self, notification, details):
+        details_obj = json.loads(details)
```

Comment:
Maybe not for this MR, but it looks like things in api.py are taking a hash, dumping it to a json string, and eventually passing it into this function which loads it from the json string back into hash. Seems like we should just pass the hash, unless I'm missing something.